### ISSUE TITLE
  Feature requests: make file splitting optional

### ISSUE THREAD

**Issue by hchaase** [duplicati/duplicati#1284]
I could be great if you included an option to disable file splitting, or setting it to a certain size.

As i am worried how I might get my files back in the event that duplicati don't work/ discontinued 10 years down the road. I would like this for better options for manual restore.

Additional information:
thank you for a great program. It is really good. it dose everything you need and is still simple. especially the quite simple data storage and and the possibility of using aescrypt and zip, gives me peace of mind, with the exception of the request mentioned above.

**Comment by hchaase** [comment]
An alternative could also be to make an option to not spilt folders in the source over several zip files.

**Comment by agrajaghh** [comment]
Thats not possible. Duplicati 2.0 isn't storing whole encrypted files. It's now storing blocks of files which can be reused. Can can't restore the original files "by hand". You can read more about it here: http://www.duplicati.com/news/theduplicati20storageengineishere

**Comment by hchaase** [comment]
I am sad to hear this. I See there is a lot of benefits for cloud storage, but it makes the backup more vulnerable. e.g. if one file is corrupted/lost it will leave a great deal of the backup useless.

And the beauty of being able to access your data without duplicati (in 1.3) is lost :( 
But I guess the choice is made. So I will have to stick with 1.3.

**Comment by agrajaghh** [comment]
There are a lot of advantages as well. You don't have to do regular full backups anymore, if just a tiny bit of a file changes, you don't have to upload the whole file again, etc...

If the remote backup data gets corrupted with duplicati 1.3 you're screwed as well...

**Comment by renestach** [comment]
It is quite difficult to discontinue a publi, open-source project. By giving you the source of Duplicati you will always be able to build a running version yourself even if technical changes are required.

When we changed the data format from 1.3 to 2.0 we saw so many technical advantages, that we intentionally dropped the "simple restore" method. This method BTW was not as simple to restore as you think as there could have been split files over multiple archives. And the system of incremental backups that are basd on previous backups is a huge risk to lose data. Just lose the first version of a file and you will be unable to restore any newer versions of it (even if the file has completely changed in the meantime).

Long story short: This will neither come for 1.3 nor for 2.0.
